README.md

```diff
@@ -64,22 +64,21 @@ See Milestones
 
 see [CHANGELOG](./CHANGELOG.md)
 
-### Usage
+## Usage
 
-- Check out this project
-- Build `cargo build`
-- `cargo run`
-- You can make changes to tests/js/test.js and build again
-- If any JS doesn't work then it's a bug! Please raise an issue
+- Clone this repo and position yourself in its folder root
```

Comment:
I think that being in the root of the project is usually implied, so you can remove this